tests/sass_stub

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env ruby
+# This program stubs out the Sass CLI. It does not apply any Sass
+# transformations and simply outputs the input file to stdout.
+
+# Version flag needs to return a value.
+if ARGV.last == '-v'
+  puts '0.1'
+else
+  # sass-spec passes in full paths to some commands. We should
+  # use the last portion of the path instead.
+  args = ARGV.map { |a| a[0] == '/' ? a.split('/').last : a }
```

Comment:
Won't this logic fail for Windows paths? Maybe a simpler thing to do would be to just look for `-I` and omit it and the following argument.

Reply:
Didn't think about! Paths omitted.